src/core/namespace.c

```diff
@@ -976,7 +976,20 @@ static int clone_device_node(const char *node, const char *temporary_mount, bool
         return 0;
 }
 
-static char *settle_runtime_dir(RuntimeScope scope) {
+static int bind_mount_device_dir(const char *dir, const char *temporary_mount) {
```

Comment:
Not a matter, but I prefer to change the order of the argument, and rename `temporary_mount` -> `prefix` or so.